File: Web/RMA-Web-Shell/projects/claimcare/src/app/claim-manager/shared/claim-care-shared/claim-invoice-container/invoice-fatal-lump-sum/fatal-lump-sum-invoice.ts
import { ClaimInvoice } from "../../../entities/claim-invoice.model";

export class FatalLumpSumInvoice {
    claimInvoiceId: number;
    description: string;
    payeeTypeId: number;
    payeeRolePlayerId: number;
    isDeleted: boolean;
    createdBy: string;
    createdDate: Date;
    modifiedBy: string;
    modifiedDate: Date;
    claimInvoice: ClaimInvoice;
    claimId: number;
}

File: Service/ClaimCare/RMA.Service.ClaimCare.Contracts/Entities/FatalLumpsumInvoice.cs
namespace RMA.Service.ClaimCare.Contracts.Entities
{
    public class FatalLumpsumInvoice
    {
        public int ClaimInvoiceId { get; set; } // ClaimInvoiceId (Primary key)
        public int PayeeId { get; set; } // PayeeId
        public int PayeeTypeId { get; set; } // PayeeTypeId
        public string Description { get; set; } // Description (length: 250)
        public string CreatedBy { get; set; } // CreatedBy (length: 50)
        public System.DateTime CreatedDate { get; set; } // CreatedDate
        public string ModifiedBy { get; set; } // ModifiedBy (length: 50)
        public System.DateTime ModifiedDate { get; set; } // ModifiedDate
        public int ClaimId { get; set; } // ClaimId
        public ClaimInvoice ClaimInvoice { get; set; }

    }
}
